langchain/audio_models/base.py

```diff
@@ -0,0 +1,22 @@
+"""Base class for text to audio models."""
+import base64
+import os
+from abc import ABC, abstractmethod
+from io import BytesIO
+
+from pydantic import BaseModel
+
+
+class AudioBase(BaseModel, ABC):
+    @staticmethod
+    def _read_mp3_audio(audio_path: str) -> str:
+        """Read audio file."""
+        assert os.path.exists(audio_path), f"Can't find file at {audio_path}"
```

Comment:
since this is validing user input, lets not use assert but rather check and raise valueerror or the like, eg
```
if not os.path.exists(audio_path):
    raise ValueError (cant find audio file)
```

source: https://google.github.io/styleguide/pyguide.html#244-decision

Reply:
nit: pref pathlib functionality over `os` package (pathlib more modern, trying to standarize on that)